notebook/static/notebook/js/shortcuteditor.js

```diff
@@ -106,27 +106,55 @@ var KeyBindingList = createClass({
              }
           }));
       });
-      childrens.unshift(createElement('div', {className:'well', key:'disclamer', dangerouslySetInnerHTML:
+      children.unshift(createElement('div', {className:'well', key:'disclamer', dangerouslySetInnerHTML:
             {__html: 
             marked(
             
-            "This dialog allows you to modify the keymap of the command mode, and persist the changes. "+
-            "You can define many type of shorctuts and sequences of keys. "+
-            "\n\n"+
-            " - Use dashes `-` to represent keys that should be pressed with modifiers, "+
-            "for example `Shift-a`, or `Ctrl-;`. \n"+
-            " - Separate by commas if the keys need to be pressed in sequence: `h,a,l,t`.\n"+
-            "\n\nYou can combine the two: `Ctrl-x,Meta-c,Meta-b,u,t,t,e,r,f,l,y`.\n"+
-            "Casing will have no effects: (e.g: `;` and `:` are the same on english keyboards)."+
-            " You need to explicitelty write the `Shift` modifier.\n"+
-            "Valid modifiers are `Cmd`, `Ctrl`, `Alt` ,`Meta`, `Cmdtrl`. Refer to developer docs "+
-            "for the corresponding keys depending on the platform."+
+            "This dialog allows you to modify the keyboard shortcuts available in command mode. "+ 
+            "Any changes will be persisted between sessions and across environments. "+
+            "You can define two kinds of shorctuts: **key combinations** and **key sequences**.\n"+
+            "\n"+
+            " - **Key Combinations**:\n"+
+            "   - Use hyphens `-` to represent keys that should be pressed at the same time.\n"+
+            "   - This is designed for use with *modifier* keys: `Cmd`, `Ctrl`, `Alt` ,`Meta`, "+
+            "`Cmdtrl`, and `Shift`.\n"+ 
+            "   - At most, one non-modifier key can exist in a key combination.\n"+
+            "   - Multiple non-modifier key can exist in a key combination.\n"+
+            "   - Modifier keys need to precede the non-modifier key in a combination.\n"+
+            "   - *Valid Examples*: `Shift-a`, `Ctrl-;`, or `Ctrl-Shift-a`. \n"+
+            "   - *Invalid Example*s: `a-b` and `a-Ctrl-Shift`. \n"+
+            " - **Key Sequences**:\n"+
+            "   - Use commas `,` to represent keys that should be pressed in sequence.\n"+
+            "   - The order in which keys must be pressed exactly matches the left-to-right order of "+
+            "the characters in the sequence, with no interruptions.\n"+
+            "     - E.g., `h,a,l,t` would be triggered by typing <kbd>h</kbd> <kbd>a</kbd> "+
+            "<kbd>l</kbd> <kbd>t</kbd> but not <kbd>h</kbd> <kbd>a</kbd> <kbd>a</kbd> <kbd>l</kbd> "+
+            "<kbd>t</kbd> or <kbd>a</kbd>  <kbd>h</kbd> <kbd>l</kbd> <kbd>t</kbd>.\n"+
+            "   - Sequences can include the same key multiple times (e.g., `d,d`).\n"+
+            "   - You cannot include a sequence that is a 'prefix' of another sequence.\n"+
```

Comment:
I read the the opposite way: `if d,d,d is bound you cannot bind d,d`. Both are true.